Riot/Managers/PushNotification/PushNotificationService.h

```diff
@@ -108,10 +108,12 @@ NS_ASSUME_NONNULL_BEGIN
 
  @param pushNotificationService PushNotificationService object.
  @param roomId Room identifier to be navigated.
+ @param userId ID of sender of the notification.
```

Comment:
threadId was missing there

Reply:
Good catch! @ismailgulek is it ok for you?